src/SelectMulti.vue

```diff
@@ -71,6 +71,15 @@
 				default: () => null
 			},
 			/**
+             * Determines whether or not typing in the input will filter the provided options.
+			 * Should be set to false if you are making API calls to fetch options.
+			 */
+            shouldFilterOptions: {
```

Comment:
I would name  it `internalSearch` like vue-multiselect has. 
https://share.getcloudapp.com/GGu2KZRW

Reply:
Updated!